apps/web/src/views/Home/components/Banners/WBNBFixedStakingBanner.tsx

```diff
@@ -0,0 +1,159 @@
+`
+
+const Desc = () => {
+  return (
+    <BannerDesc>
+      <Trans>$5,000 extra reward pool</Trans>
+    </BannerDesc>
+  )
+}
+
+const Floating = styled(FloatingGraphic)`
+  left: 3%;
+  top: 10%;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    left: 2%;
+    top: 10%;
+  }
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    left: 0;
+    top: 0;
+  }
+`
+const StyledButtonLinkAction = styled(ButtonLinkAction).withConfig({
+  shouldForwardProp: (prop) => prop !== 'showExternalIcon',
+})`
+  height: 33px;
+  border-radius: 12px;
+  padding: 0 12px;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    height: 48px;
+    border-radius: 16px;
+  }
+  & > a > svg {
+    display: ${({ showExternalIcon }) => (showExternalIcon ? 'static' : 'none')};
+  }
+`
+const Actions = styled.div`
+  position: absolute;
+  display: flex;
+  width: 100%;
+  gap: 8px;
+  z-index: 1;
+  bottom: 24px;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    position: static;
+    z-index: 0;
+  }
+`
+
+const stakeLink =
+  'https://pancakeswap.finance/simple-staking?utm_source=homepagebanner&utm_medium=website&utm_campaign=homepage&utm_id=WBNBsimplestakingcampaign'
+const learnMoreLink =
+  'https://blog.pancakeswap.finance/articles/stake-wbnb-on-pancake-swap-to-win-combo-rewards-5-000-extra-reward-pool?utm_source=homepagebanner&utm_medium=website&utm_campaign=homepage&utm_id=WBNBsimplestakingcampaign'
+
+export function WBNBFixedStakingBanner() {
+  const { t } = useTranslation()
+
+  const stakeAction = isMobile ? (
+    <StyledButtonLinkAction href={stakeLink} color="white">
+      <Flex alignItems="center" style={{ whiteSpace: 'nowrap' }}>
+        {t('Stake Now')}
+        <ArrowForwardIcon color="white" ml="4px" />
+      </Flex>
+    </StyledButtonLinkAction>
+  ) : (
+    <LinkExternalAction href={stakeLink} color="#280D5F" showExternalIcon={false}>
+      <Flex alignItems="center" style={{ whiteSpace: 'nowrap' }}>
+        {t('Stake your WBNB here')}
+        <ArrowForwardIcon color="#280D5F" ml="4px" />
+      </Flex>
+    </LinkExternalAction>
+  )
+
+  const learnMoreAction = isMobile ? (
+    <StyledButtonLinkAction href={learnMoreLink} color="white" ellipsis showExternalIcon>
+      {t('Learn More')}
+    </StyledButtonLinkAction>
+  ) : (
+    <LinkExternalAction href={learnMoreLink} color="#280D5F" style={{ whiteSpace: 'nowrap' }}>
+      {t('Learn More')}
+    </LinkExternalAction>
+  )
+
+  return (
+    <BannerContainer background="linear-gradient(64deg, rgba(214, 126, 10, 0.26) 44.37%, rgba(255, 235, 55, 0.00) 102.8%), radial-gradient(113.12% 140.14% at 26.47% 0%, #F5DF8E 0%, #FCC631 33.21%, #FF9D00 79.02%)">
+      <BannerMain
+        badges={<PancakeSwapBadge />}
+        title={
+          <BannerTitle variant="purple">
+            {isMobile ? t('Stake WBNB, Win Combo Rewards!') : t('Stake WBNB to Win Combo Rewards')}
+          </BannerTitle>
+        }
+        desc={isMobile ? null : <Desc />}
+        actions={
+          <Actions>
```

Comment:
Let's use https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/widgets-internal/components/Banner/BannerActionContainer.tsx#L4

Reply:
the `BannerActionContainer` can not use at this case, the actions should be `postion: absolute`

Reply:
why do we need position absolute?

Reply:
<img width="349" alt="image" src="https://github.com/pancakeswap/pancake-frontend/assets/144641937/083f298a-6eb4-419e-a37f-e30e64f18a11">

the actions will have vertical stack on bg